css/lists.scss

```diff
@@ -0,0 +1,46 @@
+ol, ul {
+  list-style: none;
+}
+
+ul {
+  padding: 0 !important;
+  margin: 0 !important;
+}
```

Comment:
These should go into a normalize file as they are more normalize/reset rules.
https://necolas.github.io/normalize.css/